### ISSUE TITLE
The partitioning of MIST dataset 

### ISSUE THREAD

**Issue by ccitachi** [lifangda01/AdaptiveSupervisedPatchNCE#6]
i use the MIST dataset ,but i find that the item number of trainA and valA are same. Does this mean that the patients in the training set and the test set are consistent？

**Comment by lifangda01** [comment]
Yes, and thank you for pointing that out.

I acknowledge that images from the same patient may be in both train and val, and this may potentially give rise to information leakage concerns. Since what I did is simply randomly divide the all the patches into train and val, you can put them back together and perform partitioning yourself based on the WSI IDs present in the filenames if needed.
